### PR TITLE
#139-fixing signUp button

### LINKED ISSUE #139: [BUG]:would like to fix sign up button
Styling of signUp button in not that great
Screenshot_2023-05-25-22-14-29-10_40deb401b9ffe8e1df2f1cc5ba480b12

### PR DESCRIPTION
**Pull request by Soulpurr**
## Related Issue
SignUp button taking too much padding on mobile screens

Closes: #139 

## Description of Changes
Decrease the padding of button

For example:
- Added a new function to handle XYZ.
- Updated the ABC module to fix the bug.
- Refactored code in the DEF class for improved performance.
-->

## Checklist:

- [x ] My code adheres to the established style guidelines of this project.
- [ x] I have conducted a self-review of my code.
- [x ] I have included comments in areas that may be difficult to understand.
- [x ] I have made corresponding updates to the project documentation.
- [x ] My changes have not introduced any new warnings.

## Screenshots
[scrnli_27_05_2023_13-32-26.webm](https://github.com/sourabhsikarwar/Scene-Movie-Platform/assets/92094765/05ee983f-4a23-4696-9ae7-b809cb423703)

### CODE REVIEW

**Comment by Soulpurr** [comment]
@sourabhsikarwar pls verify and approve this pr

File: src/style.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const styles = {
   Card: "h-[210px] md:h-[210px] w-[300px] md:w-[300px] hover:scale-105",
 
   button1: "bg-blue-gradient border-0 text-black px-6 py-3 rounded shadow",
-  button2: "bg-blue-gradient border-0 text-black px-4 py-2 rounded shadow",
+  button2: "bg-blue-gradient border-0 text-black px-2 py-2  rounded shadow",
```
Comment (anjula-sack) [inline src/style.js:22]:
```suggestion
  button2: "bg-blue-gradient border-0 text-black px-2 py-2 rounded shadow",
```

Comment (anjula-sack) [reply src/style.js:22]: Let's remove this extra space

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/style.js (hunk 1 above)
Comment (Soulpurr) [reply src/style.js]: @anjula-sack reomoved the extra space

**Pull request closed without merging**